### ISSUE TITLE
can not enable unity lens search ubuntu 13.04

### ISSUE THREAD

**Issue by TerrificJ** [nvbn/everpad#374]
hi, I'm working on ubuntu 13.04. From reading from google, I know lens search is default as turned on, and I also found where to set it as disabled. However, no matter I tick the checkbox of "search on the home lens" or not, still no lens icon there. How to fix that?

**Comment by TerrificJ** [comment]
yeah, it works after I reboot my computer, but I can't disable it again. I think no matter enable or disable it, it needs reboot. how can I enable or disable it without reboot?

**Comment by byaka** [comment]
i'm use 13.04 BUT with SMARTScopes.
everpad's lens doesn't work with this((
